### PR TITLE
[Crawler-Apk] Search by keyword

### PR DESCRIPTION
**Pull request by morphe157**

### CODE REVIEW

File: backend/api_v1/commands/crawler_apk/apk_font_extractor.py [hunk 1]
```diff
@@ -75,7 +76,16 @@ def find_and_save_fonts(
     for font_file in font_files:
         original_location = font_file
         temp_font_path = os.path.join(specific_font_path, os.path.basename(font_file))
-        unzip_cmd = ["unzip", "-j", "-q", apk_path, font_file, "-d", specific_font_path]
+        unzip_cmd = [
+            "unzip",
+            "-j",
+            "-qq",
```
Comment (morphe157) [inline backend/api_v1/commands/crawler_apk/apk_font_extractor.py:82]: Added it to silence annoying prompts in docker output

File: backend/api_v1/commands/crawler_apk/apk_font_extractor.py [hunk 2]
```diff
@@ -108,11 +118,45 @@ def create_csv_file(data: list[tuple[str, str, str, str]]) -> None:
 def create_zip_file() -> bytes:
     folder_path = "fonts"
     buffer = BytesIO()
-    with zipfile.ZipFile(buffer, "w", compression=zipfile.ZIP_DEFLATED) as zip_file:
+    with zipfile.ZipFile(
+        buffer, "w", compression=zipfile.ZIP_DEFLATED, strict_timestamps=False
```
Comment (morphe157) [inline backend/api_v1/commands/crawler_apk/apk_font_extractor.py:122]: `strict_timestamps=False` fixes issue with fonts which has invalid last modified day.

File: pyproject.toml [hunk 3]
```diff
@@ -13,6 +13,8 @@ python-multipart = "0.0.6"
 uvicorn = "0.20.0"
 gunicorn = "20.1.0"
 fonttools = "4.49.0"
+bs4 = "0.0.2"
```
Comment (xNykram) [inline pyproject.toml]: The latest available version of bs4 is 4.12.3, is there any reason why we used 0.0.2 here?

**Review by xNykram — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (morphe157) [reply pyproject.toml]: turns out `bs4` is just a dummy placeholder, outdated package for `beautifulsoup4`.

Switched to `beautifulsoup4` instead.

**Review by xNykram — APPROVED** [review]

**Comment by xNykram** [comment]
Pls merge it to master and inform Arek.

**Pull request merged**